.travis.yml

```diff
@@ -0,0 +1,18 @@
+dist: xenial
```

Comment:
We actually use Circle instead of travis, so if you want to get some CI going, consider putting it in https://github.com/OperationCode/operationcode-pybot/blob/staging/.circleci/config.yml